cohort-engine-api-web/src/test/java/com/ibm/cohort/engine/api/service/vt/DefaultVT.java

```diff
@@ -65,10 +77,37 @@
 public class DefaultVT extends ServiceVTBase {
 	private static final Logger logger = LoggerFactory.getLogger(DefaultVT.class.getName());
 
+	private static FhirServerConfig dataServerConfig;
+
 	@BeforeClass
 	public static void setUp() throws Exception {
 		// Instantiate service location
 		instantiateServiceLocation();
+
+		if( System.getProperty("test.host") == null ) {
+			System.setProperty("test.host", "localhost");
+		}
+
+		if(  System.getProperty("test.httpPort") == null ) {
+			System.setProperty("test.httpPort", "9089");
```

Comment:
I had this set as 9080 on my side. I don't really care what we decide on as the default, but it should be agreed upon across the team. 9080 is the default liberty port, but that obviously means that other services, such as IBM FHIR, want to also live there. There is a decent reason we might choose your port instead, but it is worth a quick conversation with the wider team.

Reply:
FYI, since it was decided that the taurus framework will be used to test these classes, the DefaultVT won't get run out on the pod and these are essentially integration tests that we can run during development time. They don't get run during the build either. 

The way I call these form eclipse is with a Launcher configuration that always passes in the following as VM args, -Dtest.host=localhost -Dtest.httpPort=9080 -Dtest.httpSslPort=9443 -Dtest.contextRoot=services/cohort -Dtest.enabledDarkFeatures=all

That being said, I would agree 9080 would be the default and would normally be overriden using the jvm arg.

Reply:
My 9080 port was already being used when I tried to start my testing, so I went with a less commonly used one. I'm fine with making it 9080 for what's committed, but that's the logic there at least.